packages/babel-plugin-import-jsx-pragma/README.md

```diff
@@ -18,14 +18,14 @@ npm install @wordpress/babel-plugin-import-jsx-pragma
 
 Refer to the [Babel Plugins documentation](http://babeljs.io/docs/en/plugins) if you don't yet have experience working with Babel plugins.
 
-Include `@wordpress/babel-plugin-import-jsx-pragma` (and [@babel/transform-react-jsx](https://babeljs.io/docs/en/babel-plugin-transform-react-jsx/)) as plugins in your Babel configuration. If you don't include both you will receive errors when encountering JSX tokens.
+Include `@wordpress/babel-plugin-import-jsx-pragma` (and [@babel/plugin-transform-react-jsx](https://babeljs.io/docs/en/babel-plugin-transform-react-jsx/)) as plugins in your Babel configuration. If you don't include both you will receive errors when encountering JSX tokens.
 
 ```js
 // .babelrc.js
 module.exports = {
 	plugins: [
 		'@wordpress/babel-plugin-import-jsx-pragma',
-		'@babel/transform-react-jsx',
+		'@babel/plugin-transform-react-jsx',
```

Comment:
I haven't checked to confirm, but I _think_ that when specifying a plugin, it is optional to include the "plugin-" prefix. 

https://babeljs.io/docs/en/options#name-normalization

I don't think it hurts to be more explicit here, though. So I'm fine with this change.